Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/details/refreshable-submission-list/RefreshableSubmissionList.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { ReactNode, useCallback } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import isNil from 'lodash/isNil';
+
+import { useAuth } from '../../../../hooks/use-auth';
+import { setCurrentPage } from '../../../../redux/features/submissionDetailsSlice';
+import { preciseFormatDate } from '../../../../utils/dates';
+import Button, { ButtonType } from '../../../guidelines/buttons/Button';
+import Heading, { HeadingType } from '../../../guidelines/headings/Heading';
+import PaginationControls from '../../../guidelines/pagination/PaginationControls';
+import SubmissionsList from '../../submissions-list/SubmissionsList';
+
+import styles from './RefreshableSubmissionList.module.scss';
+
+interface IRefreshableSubmissionListProps {
+    reload: () => void;
+
+ renderRetestButton: () => ReactNode;
+}
+const RefreshableSubmissionList = ({ renderRetestButton, reload }: IRefreshableSubmissionListProps) => {
+    const { state: { user: { permissions: { canAccessAdministration } } } } = useAuth();
+    const dispatch = useDispatch();
+    const { currentSubmission, currentSubmissionResults } =
+    useSelector((state: any) => state.submissionDetails);
```

Comment:
can't this be moved on the row above?

Reply:
with the new logic, it needs to be on 2 rows because of the max-length row